cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -231,8 +233,9 @@ def describe_cluster(cluster_name, region=None):
         cloudformation_stack_arn=cfn_stack.id,
         last_updated_time=to_utc_datetime(cfn_stack.last_updated_time),
         region=os.environ.get("AWS_DEFAULT_REGION"),
-        cluster_status=cloud_formation_status_to_cluster_status(cfn_stack.status),
+        cluster_status=cluster_status,
         scheduler=Scheduler(type=cluster.stack.scheduler, metadata=cluster.get_plugin_metadata()),
+        failure_reason=cfn_stack.get_failure_reason() if cluster_status == ClusterStatus.CREATE_FAILED else None,
```

Comment:
question, non blocking, is this actually the first time we use `failure_reason` field? Am I right saying that before this change that field would never be populated?